llvm/docs/GitHub.rst

```diff
@@ -23,6 +23,23 @@ intended to be able to support "stacked" pull-request. Do not create any branche
 llvm/llvm-project repository otherwise, please use a fork (see below). User branches that
 aren't associated with a pull-request **will be deleted**.
 
+Using Graphite for stacked Pull Requests
+========================================
+
+[Graphite](https://app.graphite.dev/) is a stacked pull request tool supported
+by the LLVM repo (the other being [reviewable.io](https://reviewable.io)).
+
+Graphite will want to create branches under `llvm/llvm-project` rather than your
+private fork, so the guidance above is critical, otherwise it will fail to
```

Comment:
```suggestion
private fork, so the guidance below is critical, otherwise it will fail to
```

Reply:
I meant the branch naming guidance, clarified.